src/components/EmailNotifications/ExistingEmailSubscription.js

```diff
@@ -79,7 +79,7 @@ function ExistingEmailSubscription({
           `}
         >
           You have previously agreed to Aragon Court's{' '}
-          <Link href="https://ant.aragon.org/legal/terms-general.pdf">
+          <Link href={`${process.env.PUBLIC_URL}/terms.pdf`}>
             legal terms
```

Comment:
same here for the email-collection.md

Reply:
and also https://github.com/aragon/protocol-dashboard/blob/develop/src/components/GlobalPreferences/Notifications/NotificationsPreferences.js#L120